cmd/scan/scan.go

```diff
@@ -63,17 +62,12 @@ func GetScanCommand(ks meta.IKubescape) *cobra.Command {
 			}
 			return nil
 		},
-		PreRun: func(cmd *cobra.Command, args []string) {
-			k8sinterface.SetClusterContextName(scanInfo.KubeContext)
-
-		},
 		PostRun: func(cmd *cobra.Command, args []string) {
 			// TODO - revert context
 		},
 	}
 
 	scanCmd.PersistentFlags().StringVarP(&scanInfo.AccountID, "account", "", "", "Kubescape SaaS account ID. Default will load account ID from cache")
-	scanCmd.PersistentFlags().StringVarP(&scanInfo.KubeContext, "kube-context", "", "", "Kube context. Default will use the current-context")
```

Comment:
Remove the `KubeContext` from the scanInfo object.
(make sure you update the httphandler pkg as well)

Reply:
I didn't remove it since I see it comes from the request. How will this be passed to the httphandler?

Reply:
The httphandler should not receive it from a request because it should scan the cluster it is running in...